lib/netsuite/actions/add.rb

```diff
@@ -77,7 +77,7 @@ def add(credentials={})
           @errors = response.errors
 
           if response.success?
-            @internal_id = response.body[:@internal_id]
+            @internal_id = response.body[:@internal_id] if response.body.respond_to? :[]
```

Comment:
Okay, that makes sense.

This is good, although we could be fancy if we wanted to and do

```rb
@internal_id = response.body.try(:[], :@internal_id)
```

which is the same thing behind the scenes.